mandiantadvantagethreatintelligence_connector.py

```diff
@@ -173,6 +175,10 @@ def _handle_test_connectivity(self, param, action_result):
         test_headers = {
             "Accept": "application/json"
         }
+        token_status = self._get_bearer_token(param, force=True)
+        if not token_status:
+            return action_result
```

Comment:
Please update the return statement to something like: return action_result.set_status(phantom.APP_ERROR, "Test Connectivity Failed")